dts/arm/st/l4/stm32l4.dtsi

```diff
@@ -234,6 +240,8 @@
 			compatible = "st,stm32-timers";
 			reg = <0x40001400 0x400>;
 			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00000020>;
+			interrupts = <55 0>;
+			interrupt-names = "global";
```

Comment:
according to the RM0394 ref Manual, IRQ 55 is for TIM 7 on stm32L43x and L44x only

Reply:
Same as commented above.